SCF.py

```diff
@@ -201,15 +198,13 @@ def calc_total_energy(Fuv_, Huv_, Duv_, Enuc_):
     Returns:
         Etot: the total energy of the molecule
     """
-    Etot= 0.5*(Duv_ * (Huv_+Fuv_)).sum()+ Enuc_
-    print (Etot)
-   
+    Etot = 0.5 * (Duv_ * (Huv_+Fuv_)).sum() + Enuc_
     """
     Replace with your implementation
 
     Should be able to implement this in one line with matrix arithmatic
 
     """
 
-
-    return Etot
\ No newline at end of file
+    return Etot
+    
```

Comment:
failing pycodestyle test on this line because it appears there is whitespace in this line, make sure that all this line contains is a new line.